index.d.ts

```diff
@@ -269,17 +269,41 @@ declare module 'plaid' {
     amount: number | null;
   }
 
+  interface ACHNumbers {
+    account: string,
+    account_id: string,
+    routing: string,
+    write_routing: string
```

Comment:
from the documentation for `Auth` [see here](https://plaid.com/docs/api/#auth) this should be `wire_routing`